solvis_graphql_api/solution_schema.py

```diff
@@ -12,6 +12,7 @@
 from nzshm_common.location.location import location_by_id
```

Comment:
could this file be deleted now?

Reply:
I think it will come back later @voj  so I'm hoarding it it :)